src/collection.js

```diff
@@ -186,3 +191,85 @@ function compareValues(a, b, descending) {
     return descending ? -1 : 1;
   }
 }
+
+function makeMapFn(projection) {
+  const id = projection && projection._id !== 0;
+  const keys = Object.getOwnPropertyNames(projection).filter(v => v !== '_id');
+  const passthroughfn = generatePassThroughFN(id, keys);
```

Comment:
If you're going to use "fn", please keep the case consistent. You use "fn", "FN", and "Fn".